dataikuapi/dssclient.py

```diff
@@ -966,7 +978,7 @@ def execute(self, settings = {}):
         @warning: You must check the 'success' flag
         """
         # Empty JSON dicts can't be parsed properly
-        if settings == {}:
-            settings["_"] = "_"
+        if settings is None:
```

Comment:
In that case, if the user actually passes `sesttings={}`, the behaviour changes

Reply:
Nice catch!